### PR TITLE
Add usage metrics page

### LINKED ISSUE #2391: Add a section around Usage Metrics and their definition
We will need a definition of our usage metrics and their definition.
In C7 docs we have something similar like: https://docs.camunda.org/manual/7.19/user-guide/process-engine/metrics/

From this page we should then link to SaaS and SM ways of finding out about the metrics, e.g. Billing Page in Console vs. currently API way in SM.

### PR DESCRIPTION
**Pull request by akeller**
## Description

Adds metric page via https://docs.google.com/document/d/1Fre_Nb51NfwiKyBmv0Ue9sOuJH0eMAcB3ZhVq-DjqkQ/edit. Closes #2391

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/reference/usage-metrics.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+id: usage-metrics
+title: "Usage metrics"
+description: "Learn about the three main usage metrics that impact Camunda Platform 8 pricing."
+---
+
+There are three main usage metrics that have an impact on Camunda Platform 8 pricing. It is important to understand these metrics and how to retrieve them.
```
Comment (christinaausley) [inline docs/reference/usage-metrics.md:7]: > It is important to understand these metrics and how to retrieve them.

I can already answer this, but a user might as *why* it is important to understand these metrics. Could be as simple as adding a word or two for purpose.

File: docs/reference/usage-metrics.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Definition of metrics
+
+### Root process instance
+
+The number of **root process instance** executions started. This is also known as process instances (PI). A **root process instance** has no parent process instance, i.e. it is a top-level execution.
+
+### Decision instance
+
+The number of evaluated **decision instances** (DI). A **decision instance** is a DMN decision table or a DMN literal expression. In a DRD each evaluated decision table or expression is counted separately.
```
Comment (christinaausley) [inline docs/reference/usage-metrics.md:17]: Is it worth linking to process instance details, DMN, or task user documentation for clarity, or is it assumed if a user is visiting this page, they should be aware of these concepts?

**Review by christinaausley — DISMISSED** [review]
Two non-blockers 👍 Adjusting a typo in my subsequent commit!

**Review by felix-mueller — APPROVED** [review]

**Comment by akeller** [comment]
Thanks for the quick review @christinaausley! I mostly just C&P'd these changes over from the internal working doc. I'll incorporate your feedback and backport these changes later today.

File: docs/reference/usage-metrics.md (hunk 1 above)
Comment (akeller) [reply docs/reference/usage-metrics.md:7]: > It is important to understand these definitions, their impact on billing, and how to retrieve them.

Added in an incoming commit. WDYT?

File: docs/reference/usage-metrics.md (hunk 2 above)
Comment (akeller) [reply docs/reference/usage-metrics.md:17]: This is a good question - I'm not sure. Let's add a few links and iterate in the future if we get any feedback.

File: docs/reference/usage-metrics.md (hunk 1 above)
Comment (christinaausley) [reply docs/reference/usage-metrics.md]: Awesome 🚀

File: docs/reference/usage-metrics.md (hunk 2 above)
Comment (christinaausley) [reply docs/reference/usage-metrics.md]: 👍

**Review by christinaausley — DISMISSED** [review]

**Review by christinaausley — APPROVED** [review]
Backpot approved 👍

**Pull request merged**